client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx

```diff
@@ -118,31 +100,23 @@ const WooConfirm: Step = function WooCommerceConfirm( { navigation } ) {
 	const upgradingPlan = productName ? productsList?.[ productName ] : null;
 
 	// Filter the Woop transferring blockers.
-	const transferringBlockers = eligibilityHolds?.filter(
-		( hold: TransferEligibilityError ) => ! TRANSFERRING_NOT_BLOCKERS.includes( hold.code )
-	);
+	const transferringBlockers =
+		eligibilityHolds?.filter( ( hold: string ) => ! TRANSFERRING_NOT_BLOCKERS.includes( hold ) ) ??
+		[];
 
 	const isTransferStuck = latestAtomicTransfer?.is_stuck || false;
 	const isBlockByTransferStatus = latestAtomicTransferError || false;
 
 	// Add blocked-transfer-hold when something is wrong in the transfer status.
 	if (
-		! transferringBlockers?.includes( {
-			code: eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER,
-			message: '',
-		} ) &&
+		! transferringBlockers.includes( eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER ) &&
 		( isBlockByTransferStatus || isTransferStuck )
 	) {
-		transferringBlockers?.push( {
-			code: eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER,
-			message: '',
-		} );
+		transferringBlockers.push( eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER );
```

Comment:
There is a logic error here. Previously, when it was possible for `transferringBlockers` to be null, this error would not have been pushed to the array, even though it probably should be. The ultimate change was to make sure that `transferringBlockers` was initialized to an array at least.